helix-core/src/lib.rs

```diff
@@ -48,15 +51,26 @@ pub fn find_root(root: Option<&str>) -> Option<std::path::PathBuf> {
 
 #[cfg(not(embed_runtime))]
 pub fn runtime_dir() -> std::path::PathBuf {
-    // runtime env var || dir where binary is located
-    std::env::var("HELIX_RUNTIME")
-        .map(|path| path.into())
-        .unwrap_or_else(|_| {
-            std::env::current_exe()
-                .ok()
-                .and_then(|path| path.parent().map(|path| path.to_path_buf()))
-                .unwrap()
-        })
+    if let Ok(dir) = std::env::var("HELIX_RUNTIME") {
+        return dir.into();
+    }
+
+    const RT_DIR: &str = "runtime";
+    let conf_dir = config_dir().join(RT_DIR);
+    if conf_dir.exists() {
+        return conf_dir;
+    }
+
+    if let Ok(dir) = std::env::var("CARGO_MANIFEST_DIR") {
```

Comment:
Nice! This is only via 'cargo' right?

Reply:
Yes, it’s exported only during cargo build/run.

Reply:
Good idea. I didn't thought about this, if we run from cargo we don't need to use the runtime dir with this.

Reply:
Good idea. I didn't thought about this, if we run from cargo we don't need to use the runtime dir with this.